index.html

```diff
@@ -340,5 +340,33 @@ <h1>
         </div>
       </div>
     </section>
+    <div class="form-div">
+      <form action="https://formspree.io/f/xbjboedq" method="POST" class="form">
+        <h2>Contact me</h2>
+
+        <p>
+          If you have an application you are interested in developing, a feature
+          that you need built or a project that needs coding. I’d love to help
+          with it
+        </p>
+        <input
+          type="text"
+          maxlength="30"
+          placeholder="Enter your name here"
+          required
+        />
+        <input type="email" placeholder="Enter your email here" required />
+        <textarea
+          name="text-area"
+          id="text-area"
+          cols="20"
+          rows="7"
+          maxlength="500"
+          placeholder="Write your message here"
+          required
+        ></textarea>
+        <button type="submit">Get In Touch</button>
+      </form>
+    </div>
```

Comment:
[Optional] Good job on implementing all of the requested changes. As you know it's highly recommended to stick with the Figma template as much as possible. So it's recommended to add the bottom line shape to your page. 😊

![image](https://user-images.githubusercontent.com/34779928/192635134-258fbe32-66f5-4761-8577-a7f6c94edfbe.png)

Reply:
Fixed and thank you very much